test/Swagger.ObjectModel.Tests/Builders/InfoBuilderTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Swagger.ObjectModel.Builders;
+using Xunit;
+
+namespace Swagger.ObjectModel.Tests.Builders
+{
+    public class InfoBuilderTest
+    {
+        [Theory]
+        [InlineData("", "")]
+        [InlineData("", "1.0")]
+        [InlineData("title", "")]
+        [InlineData(null, null)]
+        [InlineData(null, "1.0")]
+        [InlineData("title", null)]
+        public void Should_ThrowRequiredFieldException_WhenTitleOrVersionIsEmptyOrWhiteSpace(string title, string version)
+        {
+            Assert.Throws(typeof(RequiredFieldException), () => new InfoBuilder(title, version).Build());
+        }
+
+        [Theory]
+        [InlineData("title", "1.0","description")]
+        [InlineData("title", "1.0", "")]
+        public void Should_AbleToSetDescription(string title, string version,string descrption)
```

Comment:
Minor formatting: 
You forgot a space: `version,string` 
You also missed one two lines above: `"1.0","description"`

You should double check the rest of the files too.

Reply:
It's probably worth us integrating a linter in our build process for style things like this, I expect there will be lots of minor fixes across the whole repo if we do get that up and running so I'm fine considering this non-blocking (though I would prefer if it gets fixed first!)